app/src/main/java/com/hunguet123/searchImage/data/source/local/room/PhotoRoomDatabase.kt

```diff
@@ -0,0 +1,10 @@
+package com.hunguet123.searchImage.data.source.local.room
+
+import androidx.room.Database
+import androidx.room.RoomDatabase
+import com.hunguet123.searchImage.data.model.Photo
+
+@Database(entities = [Photo::class], version = 1, exportSchema = false)
+abstract class PhotoRoomDatabase : RoomDatabase() {
+    abstract fun PhotoLocalDao() : PhotoLocalDao
```

Comment:
đặt tên hàm đúng quy tắc, mà database là local rồi thì không cần thiết phải thêm chữ local vào nữa 
remote => API, firebase, ... 
local => Database, shared Prefs, files, ...  
```suggestion
    abstract fun photoDao() : PhotoLocalDao
```